desktop/core/requirements.txt

```diff
@@ -44,6 +44,7 @@ Markdown==3.1
 mysqlclient==1.4.6
 nose==1.3.7
 openpyxl==2.6.2
+pandas==1.3.5
```

Comment:
`pandas==1.3.5` requires Py3.7+

Good to change to `1.1.5`? That requires Py3.6+ and we can add pandas Py2 package in ext-py.

Or maybe try `0.24.2`? which requires `Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*`

Reply:
Better to try with #2 first if it contains the things we need I guess.